src/SME.SGP.Dominio.Servicos/ServicoCompensacaoAusencia.cs

```diff
@@ -368,7 +370,10 @@ public async Task Excluir(long[] compensacoesIds)
                     unitOfWork.Rollback();
                 }
             }
-
+            foreach (var item in listaCompensacaoDescricao)
+            {
+                await mediator.Send(new DeletarArquivoDeRegistroExcluidoCommand(item,TipoArquivo.CompensacaoAusencia.Name()));
+            }
```

Comment:
da uma protegida nesse cara: listaCompensacaoDescricao
if(listaCompensacaoDescricao != null || listaCompensacaoDescricao.Any())
        foreach (var item in listaCompensacaoDescricao)
            {
                await mediator.Send(new DeletarArquivoDeRegistroExcluidoCommand(item,TipoArquivo.CompensacaoAusencia.Name()));
            }